src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -16,6 +16,22 @@
  * @param {( AuthenticatorEnrollment[] || Authenticator[] )} authenticators
  */
 const createAuthenticatorOptions = (options = []) => {
+  // Split OV into individual entries (one for each method).
```

Comment:
Should this be in formInputFactory createAuthenticatorVerifySelectView? Similar to webauthn?

Reply:
Yeah we can do it in either place.